proposals/2021-04-lower-deposit-requirement/README.md

```diff
@@ -0,0 +1,53 @@
+# Parameter change: lower minimum proposal deposit amount
+
+## Summary
+
+The current deposit amount of 512 ATOMs prohibits valuable governance activity from small holders or those with most of their ATOM staked. We propose lowering the requirment to 64 ATOMS.
+
+## Objectives
+
+1. Enable community members with good ideas but little capital to participate in governance and request resources from the community pool treasury.
+2. Improve the governance UX for holders who keep most of their ATOM staked.
+3. Increase utilization of treasury (currently 666,457 ATOM, approximately $14 MM USD, at time of writing).
+4. Accelerate Cosmos Hub development and growth.
+
+## Background
+
+Current deposit is 512 ATOMs (approximately $10k USD today). The ATOM price when the community treasury was activated (2019-05-03) was $4.99 (source: CoinMarketCap), meaning the total required deposit to submit a proposal was $2,555. Today, most proposers must coordinate with large ATOM holders to request additional funds in order to meet the minimum deposit requirements. This also applies to large ATOM holders who want to be active in governance but do not have enough liquid ATOM to meet the deposit requirements, as staked ATOM cannot be used to post deposits.
```

Comment:
While this is good motivation, we'll come across the same barrier to entry if the price of the ATOM token fluctuates greatly. What is really needed is an oracle to allows the minimum deposit to fluctuate and become elastic depending on the current price.

However, for the time being, this is sufficient.

Reply:
Totally agree, this is very much a temporary solution. I can indicate that in the proposal.